src/uu/ls/src/ls.rs

```diff
@@ -1205,6 +1240,7 @@ only ignore '.' and '..'.",
 /// Represents a Path along with it's associated data
 /// Any data that will be reused several times makes sense to be added to this structure
 /// Caching data here helps eliminate redundant syscalls to fetch same information
+#[derive(Debug, Clone)]
```

Comment:
Is this leftover from some experimentation?

Reply:
Yes, caught me!  Thank you.